src/js/components/put-away/PutAwaySecondPage.jsx

```diff
@@ -131,21 +131,36 @@ class PutAwaySecondPage extends Component {
       Cell: (props) => {
         const itemIndex = props.index;
         const edit = _.get(this.state.putAway.putawayItems, `[${itemIndex}].edit`);
+        let disabledMessage;
+
+        if (props.value < 1) {
+          disabledMessage = this.props.translate(
+            'react.putAway.negativeQuantity.label',
+            'Quantity cannot be lower than 1',
+          );
+        }
+
+        if (props.original && props.value > props.original.quantityAvailable) {
+          disabledMessage = this.props.translate(
+            'react.putAway.higherQuantity.label',
+            'Quantity cannot be higher than original putaway item quantity',
+          );
+        }
+
         if (edit) {
           return (
             <Tooltip
-              html={this.props.translate(
-                'react.putAway.higherQuantity.label',
-                'Quantity cannot be higher than original putaway item quantity',
-              )}
-              disabled={props.value <= props.original.quantityAvailable}
+              html={disabledMessage}
+              disabled={!disabledMessage}
               theme="transparent"
               arrow="true"
               delay="150"
               duration="250"
               hideDelay="50"
             >
-              <div className={props.value > props.original.quantityAvailable ? 'has-error' : ''}>
+              <div className={props.value > props.original.quantityAvailable ||
```

Comment:
I'd split this logic to a variables:
```
const isQuantityHigherThanOriginal = props?.value > props.original?.quantityAvailable
const isQuantityLowerThan1 = props?.value < 1
```
and use it above and also here inside `<div>`
```
<div className={(isQuantityHigherThanOriginal || isQuantityLowerThan1) ? 'has-error' : ''}>
```